code/game/objects/structures/crates_lockers/closets/secure/scientist.dm

```diff
@@ -1,5 +1,5 @@
 /obj/structure/closet/secure_closet/research_director
-	name = "research director's locker"
+	name = "Research Director's locker"
```

Comment:
i really don't think you should change any of these that are referring to a job. we don't do this normally and it looks terrible

Reply:
ah this was san's complaint, i did not realize. i thought he was only concerned about the cells